packages/modal/README.md

```diff
@@ -0,0 +1,103 @@
+# TSDX User Guide
```

Comment:
please update read me file, put instructions here how to install and use package